cli/src/lib.rs

```diff
@@ -149,23 +151,29 @@ impl NodeBuilder {
         self
     }
 
+    /// Adds a service instance that will be available immediately after creating a genesis block.
+    ///
+    /// Make sure that artifact for this instance is provided to this builder.
+    pub fn with_default_instance(mut self, instance: impl Into<InstanceInitParams>) -> Self {
+        self.default_instances.push(instance.into());
+        self
+    }
+
+    /// Adds a default Rust service instance that will be available immediately after creating a
+    /// genesis block.
+    pub fn with_default_service(self, service: impl DefaultInstance) -> Self {
```

Comment:
I'd suggest to use `with_default_rust_service` for uniformity.

Reply:
To uniformity we should also rename `with_service` to `with_rust_service` but this is breaking change.